.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Create GitHub Release
+
+on:
+  # Run on demand
+  workflow_dispatch:
+
+  # Run on push to main when .bumpversion.cfg version is updated
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - .bumpversion.cfg
+
+jobs:
+  release:
+    uses: plus3it/actions-workflows/.github/workflows/release.yml@15ec2909fc026e469664a5978332be1741fbabf3
```

Comment:
This will need to pass in the option to skip the mockstack test...

Reply:
Disabled mockstack test. Thanks